### PR TITLE
from the guy on so =)

### PR DESCRIPTION
**Pull request by Androz2091**

### CODE REVIEW

**Comment by JavascriptLearner815** [comment]
This has already been commited and merging would conflict right now. I'll just go ahead and close it.

**Comment by Androz2091** [comment]
Right np ^^

**Comment by JavascriptLearner815** [comment]
Approved.

**Review by JavascriptLearner815 — COMMENTED** [review]
The code has been changed and this is approved.

File: code/commands/has-permission.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ module.exports = {
             'MANAGE_WEBHOOKS',
             'MANAGE_EMOJIS'
         ];
-        const member = message.mentions.users.first();
+        const member = message.mentions.members.first();
```
Comment (JavascriptLearner815) [inline code/commands/has-permission.js:53]: This is approved and added.

**Review by JavascriptLearner815 — COMMENTED** [review]
All of this is approved.

**Pull request closed without merging**